src/components/autocomplete-dropdown/autocomplete-dropdown.test.tsx

```diff
@@ -65,10 +65,20 @@ describe('autocomplete-dropdown', () => {
   });
 
   it('it renders match suggestions', async () => {
-    findMatches = () => { return [
-      { id: 'result-first-id', value: 'result-first-value', route: 'result-first-route' },
-      { id: 'result-second-id', value: 'result-second-value', route: 'result-second-route' },
-    ]};
```

Comment:
haha. yeah. .. i think with nested objects like this, it's less important that the inner objects are on separate lines, as you're mostly just going to be adding/removing single objects, rather than touching properties in every one of them. wonder if it can be based on number of properties.

Reply:
@mrval5 is it possible to have it based on number of properties? eg. .. if 3 or less, allow single line, otherwise make it multiple lines?